### ISSUE TITLE
需要翻译！need translation！

### ISSUE THREAD

**Issue by yan-forever** [google/UIforETW#156]
需要中文，需要多语言支持。Need Chinese, need multi-language support.

**Comment by randomascii** [comment]
I agree. The codebase is at least Unicode, but no hooks for translation have been added. I'd be willing to help do this, but I don't think I will do it on my own.
